en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -314,90 +537,126 @@ class Page extends SiteTree
 
 This section deals with specialised operations that can be performed on versioned DataObjects.
 
-### Reading latest versions by stage
+### Reading versions by stage
+
+By default, all records are retrieved from the "Stage" (aka draft) stage, which pulls from the same database table you would be using if there were no stages or versions at all.
 
-By default, all records are retrieved from the "Draft" stage (so the `MyRecord` table in our example). You can
-explicitly request a certain stage through various getters on the `Versioned` class.
+You can explicitly request a specific stage through various static methods on the `Versioned` class.
 
 ```php
-<?php
 use SilverStripe\Versioned\Versioned;
 
 // Fetching multiple records
-$stageRecords = Versioned::get_by_stage('MyRecord', Versioned::DRAFT);
-$liveRecords = Versioned::get_by_stage('MyRecord', Versioned::LIVE);
+$stageRecords = Versioned::get_by_stage(MyRecord::class, Versioned::DRAFT);
+$liveRecords = Versioned::get_by_stage(MyRecord::class, Versioned::LIVE);
 
 // Fetching a single record
-$stageRecord = Versioned::get_by_stage('MyRecord', Versioned::DRAFT)->byID(99);
-$liveRecord = Versioned::get_by_stage('MyRecord', Versioned::LIVE)->byID(99);
+$stageRecord = Versioned::get_by_stage(MyRecord::class, Versioned::DRAFT)->byID(99);
+$liveRecord = Versioned::get_by_stage(MyRecord::class, Versioned::LIVE)->byID(99);
 ```
 
+You can also change what stage is being used for queries by calling [`Versioned::set_stage()`](api:SilverStripe\Versioned\Versioned::set_stage()) - but make sure you set the reading mode back to what it was before you started!
+
+```php
+use SilverStripe\Versioned\Versioned;
+
+// Temporarily store the current mode before setting the mode we want to use
+$oldMode = Versioned::get_reading_mode();
+Versioned::set_stage(Versioned::LIVE);
+
+$liveRecords = MyRecord::get();
+
+// Don't forget to set the mode back afterwards!
+Versioned::set_reading_mode($oldMode);
+```
+
+[info]
+`Versioned::set_stage(Versioned::LIVE)` is the equivalent of `Versioned::set_reading_mode('Stage.' . Versioned::LIVE)`.
+[/info]
+
+That said, it is better practice to use [`Versioned::withVersionedMode()`](api:SilverStripe\Versioned\Versioned::withVersionedMode()) because this removes the possibility of forgetting to set the stage back to what it was afterward.
```

Comment:
Should provide the better alternative as the first example. Then you can provide the old style example as a second example, if at all.

Reply:
Done